src/Console/Commands/PostmanCollection.php

```diff
@@ -0,0 +1,201 @@
+    {
+        $modulePath = $this->getModulePath();
+
+        $this->postmanFiles = array_filter(glob($modulePath . '/*/' . self::DOCS_FOLDER_NAME . '/*.postman.json'));
+
+        $this->info('Success Load Postman Files ..');
+    }
+
+    /**
+     * Load Content of postman file
+     * 
+     * @return void
+    */
+    private function loadContentPostmanFiles()
+    {
+        foreach($this->postmanFiles as $postmanFile) {
+            $content = $this->pareFileToJson($postmanFile);
+
+            $name = $content['info']['name'];
+            
+            $this->filesContent[] = $content;
+
+            $this->info("Collection [ $name ] was successfully added.");
+        }
+    }
+
+    /**
+     * Create Root Collection
+     * 
+     * @return void
+    */
+    private function createCollection()
+    {
+        $appName = env('APP_NAME', 'Postman Collection with Mongez');
+
+
+        $collection = [
+            'item' => [],
+            'variable' => [],
+            'info' => [
+                'name' => $appName,
+                'schema' => 'https://schema.getpostman.com/json/collection/v2.1.0/collection.json'
+            ]
+        ];
+
+        foreach($this->filesContent as $content) {
+            // Remove Content of Globel Variable in each Collection  
+            // will use it in Root Collection Only
+            $content['variable'] = [];
+
+            $content['name'] = $content['info']['name'];
+
+            $collection['item'][] = $content;
+        }
+
+        $collection['variable'] = $this->getGlobalVariables();
+
+        $fileName = $this->optionHasValue('fileName') ? $this->option('fileName') : self::DEFAULT_GENERATE_FILE_NAME;
+
+        file_put_contents(base_path("$fileName.postman.json"), json_encode($collection, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE));
+
+        $this->info("Success Generate Postman Collection for ' $appName '");
+    }
+
+    /**
+     * Get Global Variables
+     * 
+     * @return array
+    */
+    private function getGlobalVariables()
+    {
+        $allGlobalVariables = [];
+
+        foreach(array_column($this->filesContent, 'variable') as $globalVariable) {
+            foreach($globalVariable as $variable){
+                $allGlobalVariables[$variable['key']] = $variable;
+            }
+        }
+        
+        return $allGlobalVariables;
+    }
+
+
+    /**
+     * parsing file from text to json
+     * 
+     * @param string $file
+     * @return array
+    */
+    private function pareFileToJson($file)
+    {
+        $content = file_get_contents($file);
+
+        return json_decode($content, true);
+    }
+    
+    /**
+     * Get Module Path 
+     * 
+     * @return string
+    */
+    private function getModulePath()
+    {
+        return base_path(self::MODULE_PATH);
+    }
+
+    /**
+     * Return Module Repo
+     * 
+     * @return mixed
+    */
+    public function getRepositry($repo)
```

Comment:
Why this method exists?, if you want to get a repository use `repo` function instead, or inject the RepositoryTrait.

Reply:
sorry, this function is not important right now. Removed.